commands/give.js

```diff
@@ -1,11 +1,37 @@
+const update_dabs = require("./_update_dabs.js")
+const pay_dabs = require("./_pay_dabs.js")
+const get_dabs = require("./_get_dabs.js")
+
 module.exports = {
     title: "Give",
-    desc: "Complain to @March#7572 for this not being implemented yet.",
-    syntax: "`TODO`",
+    desc: "Give <amount> dabs each to each user mentioned. @everyone or @here"
+    + " will cancel the transfer",
+    syntax: "`{prefix}give <amount> [mentions...]`",
     alias: ["give"],
     owner_only: false,
     affect_config: false,
     action: function(message, config) {
-        message.channel.send("Try `$help give`.")
+        amount = message.content.split(" ")[1]
```

Comment:
If @everyone and @here cancel the command. Maybe you can still make it work for roles.
This is completely optional.